about.js

```diff
@@ -21,4 +21,11 @@ document.querySelector('.summits-holder').innerHTML = summits.map((summit) => `
               <span class="px-1 summit-message">${summit.message}</span>
           </div>
       </div>
-  `).join('');
\ No newline at end of file
+  `).join('');
+
+function ReadElement(selector) {
+  return document.querySelector(selector);
+}
+const aboutBtn = ReadElement('#about-desktop-nav');
+aboutBtn.classList.add('primary-color');
+aboutBtn.classList.remove('desktop-nav-item');
```

Comment:
- [ ] Apart from the logo, the about section those not lead back to the home page. Kindly implement this section for better accessibility of your website 👍 

## View:
![image](https://user-images.githubusercontent.com/50990925/227094314-e886c553-a371-4422-b3d2-9003ad5495af.png)

Reply:
done 👍